sensu-report

```diff
@@ -225,10 +256,13 @@ def main():
         options.server, options.port, options.client,
         options.user, options.password)
     print_silence_report(silence_data, options.max_line_length)
+    http_code = check_client_exists(
+        options.server, options.port, options.client,
+        options.user, options.password)
     events = fetch_sensu_data(
```

Comment:
If we know that a client doesn't exist, I would prefer that we don't even bother checking for events.

```python
if client_exists:
    events =...
    print_report()
else:
   print_client_not_found()
```